app.json

```diff
@@ -53,7 +53,15 @@
 			"backgroundColor": "#00000000"
 		},
 		"plugins": [
-			"sentry-expo"
+			"sentry-expo",
+			[
+				"@viro-community/react-viro",
```

Comment:
i got it working, but this is still warned about by vscode for some reason

![Bildschirmfoto 2022-07-13 um 15 19 12](https://user-images.githubusercontent.com/1942953/178759135-a4f3abe8-4951-4664-bfdb-75d7fddfd8d9.png)

Reply:
Actually, this error should go away when you install the `viro` package.